clickhouse-grpc-client/src/main/java/com/clickhouse/client/grpc/ClickHouseGrpcChannelFactory.java

```diff
@@ -193,9 +193,6 @@ protected void setupMisc() {
             builder.enableFullStreamDecompression();
         }
 
-        if (config.isUseNoProxy()) {
```

Comment:
Let's simply replace `config.isUseNoProxy()` to `config.getProxyType() == ClickHouseProxyType.DIRECT`.